.github/workflows/soaking.yml

```
@@ -204,15 +204,15 @@ jobs:
         run: |
           version=$(echo "${{ steps.extract-sdk-layer-arn.outputs.stdout }}" | cut -d : -f 8)
           echo "Found version number: $version"
-          echo "::set-output name=${{ matrix.language }}-${{ matrix.instrumentation-type }}-version::$version"
+          echo ""${{ matrix.language }}-${{ matrix.instrumentation-type }}-version=$version" >> $GITHUB_OUTPUT"
```

```suggestion
          echo "${{ matrix.language }}-${{ matrix.instrumentation-type }}-version=$version" >> $GITHUB_OUTPUT
```
I believe double quote should only wrap the string and not the redirect. See [this](https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/) blog for reference also.